### PR TITLE
[FIX] - Use the new location of the Mint API Key

### PR DESCRIPTION
**Pull request by burkematthew**
This is the next part of a series of fixes required based on the new layout and flow of Mint.  In the update, their scripts changed and so the location of the API Key changed.  

NOTE: This doesn't completely fix the problems that users are experiencing, but is one necessary step in the process.

### CODE REVIEW

File: mintapi/api.py [hunk 1]
```diff
@@ -144,7 +144,7 @@ def get_rnd(cls):  # {{{
         ).zfill(3)
 
     def _get_api_key_header(self):
-        key_var = "window.MintConfig.browserAuthAPIKey"
+        key_var = "__shellInternal.OILConfigs.key"
```
Comment (mrooney) [inline mintapi/api.py:147]: That is really something!

**Review by mrooney — APPROVED** [review]

**Comment by tsdg112** [comment]
I have several Mint accounts.  Some of them are still using the javascript-user flow and work fine with the current Mintapi.  Will this change break retrieving data from the accounts that have not yet been converted?

**Comment by mrooney** [comment]
> I have several Mint accounts. Some of them are still using the javascript-user flow and work fine with the current Mintapi. Will this change break retrieving data from the accounts that have not yet been converted?

That's a great question! In the ideal world it would be great if the new endpoints worked for all accounts so it didn't matter. Perhaps you can test before a release and we can verify. @burkematthew let me know if you have any guesses one way or the other.

**Comment by burkematthew** [comment]
@mrooney @tsdg112 I'd imagine that they're simply rolling out the UI to phases of users and it's only a matter of time before all users have the new UI. The endpoints should have no problem working for those users even that still have the old UI.

I do agree that it'd be good to have you test before the next release to verify all is working as expected for you on the old flow. In the meantime, we are going to be pushing forward to get the new flow working.

**Comment by mrooney** [comment]
@burkematthew sounds good! I think in a worst case, in the interim we can always say to install the older version from pypi if your account isn't rolled out yet.

**Pull request merged**